composites/Plugin/Shared/tests/YoastButtonTest.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { YoastButton } from "../components/YoastButton";
+
+test( "the YoastButton matches the snapshot", () => {
+	const component = renderer.create(
+		<YoastButton backgroundColor="#c00" textColor="#fff">ButtonValue</YoastButton>
+	);
+
+	let tree = component.toJSON();
+	expect( tree ).toMatchSnapshot();
+} );
+
+test( "the YoastButton executes callback", () => {
+	const component = renderer.create(
+		<YoastButton onClick={
+			() => {
+				return "clicked";
+			}
+		}>ButtonValue</YoastButton>
+	);
+
+	let tree = component.toJSON();
+	expect( tree ).toMatchSnapshot();
+
+	tree.props.onClick();
+
+	tree = component.toJSON();
```

Comment:
As discussed during the stand-up. There seems to be no difference in the created snapshot, so I don't think this test adds anything.

Reply:
Not arguing, just asking because I really don't know how this works behind the scenes. I see this same pattern is used in `ButtonTest.js` and a few times also on my-yoast, there are a few occurrences there, see for example things like this one:
```
	// manually trigger the callback
	tree.props.onClick();
	
	// re-rendering
	tree = component.toJSON();
	expect(tree).toMatchSnapshot();
```

@atimmer can we have a quick feedback, when you have a chance? Thanks!